examples/golden_gate_assembly.py

```diff
@@ -30,15 +30,15 @@
 #############################################
 # Create the protocol
 print("Creating protocol")
-protocol = labop.Protocol("GoldenGate_assembly")
-protocol.name = "Golden Gate Assembly"
-protocol.description = """
+activity = labop.Protocol("GoldenGate_assembly")
```

Comment:
What's motivating this change in convention, from calling it a protocol to an activity?

I am fine with this change in convention, but would like to note that for communication and presentation purposes (for example, if I were including a code example in a manuscript about protocols), I would generally prefer to use `protocol`

Reply:
This was in error